frontend/packages/kubevirt-plugin/integration-tests/tests/models/editFlavorView.ts

```diff
@@ -2,7 +2,7 @@ import { $ } from 'protractor';
 
 export const flavorDropdownId = '#vm-flavor-modal-flavor-dropdown';
 
-export const modalTitle = () => $('.modal-title');
+export const modalTitle = () => $('.pf-c-title');
```

Comment:
This is a bit generic.  Perhaps it'd be better to add something like `data-test-id="modal-title"` to `modal.tsx`.

Reply:
Yeah, do like that better